test/pregel_test.go

```diff
@@ -0,0 +1,77 @@
+//
+// DISCLAIMER
+//
+// Copyright 2022 ArangoDB GmbH, Cologne, Germany
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// Copyright holder is ArangoDB GmbH, Cologne, Germany
+//
+
+package test
+
+import (
+	"context"
+	"testing"
+
+	"github.com/arangodb/go-driver"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestCreatePregelJob(t *testing.T) {
+	ctx := context.Background()
+	c := createClientFromEnv(t, true)
+	skipBelowVersion(c, "3.10", t)
```

Comment:
Probably these tests should executed only at EE versions? The Arangodb docs says pregel works only with smart graphs, which are available only in EE

Reply:
it works on Community version as well :)